system_metrics_collector/test/system_metrics_collector/test_linux_process_cpu_measurement_node.cpp

```diff
@@ -27,26 +27,31 @@
 #include "metrics_statistics_msgs/msg/metrics_message.hpp"
 #include "metrics_statistics_msgs/msg/statistic_data_type.hpp"
 
+#include "moving_average_statistics/moving_average.hpp"
+
 #include "system_metrics_collector/constants.hpp"
 #include "system_metrics_collector/linux_process_cpu_measurement_node.hpp"
 #include "system_metrics_collector/proc_cpu_data.hpp"
 #include "system_metrics_collector/utilities.hpp"
 
 #include "test_constants.hpp"
+#include "test_functions.hpp"
 
 using lifecycle_msgs::msg::State;
 using metrics_statistics_msgs::msg::MetricsMessage;
 using metrics_statistics_msgs::msg::StatisticDataPoint;
 using metrics_statistics_msgs::msg::StatisticDataType;
+using moving_average_statistics::MovingAverageStatistics;
 using moving_average_statistics::StatisticData;
+using test_constants::kProcPidSamples;
```

Comment:
It doesn't matter much since this is a `.cpp` file, but these can be moved into the anonymous namespace to explicitly restrict their scope to this file.